src/behapy/config.py

```diff
@@ -0,0 +1,28 @@
+'''
+lydia barnes
+lydia.barnes@tuta.io
+
+created november 2022
+
+example parameters for fibre photometry project
+
+'''
+
+def getParameters():
```

Comment:
Could we define the parameters as a json or yaml file, and create a `get_parameters` type function to read the config from somewhere?

Reply:
have done this but not tested -- popped get_parameters under utils.py